controllers/ibmvpccluster_controller.go

```diff
@@ -170,12 +216,25 @@ func (r *IBMVPCClusterReconciler) reconcileDelete(clusterScope *scope.ClusterSco
 		return ctrl.Result{}, nil
 	}
 
+	if clusterScope.IBMVPCCluster.Spec.ControlPlaneLoadBalancer != nil {
+		lbCount, err := clusterScope.DeleteLoadBalancer()
+		if err != nil {
+			return ctrl.Result{}, errors.Wrap(err, "failed to delete loadBalancer")
+		}
+		// skip deleting other resources if still have loadBalancers running.
+		if lbCount != int(0) {
+			return ctrl.Result{}, nil
```

Comment:
do you want to add `RequeueAfter` here to check again after few minutes rather than waiting for default sync up? or its not really needed because we still have finalizer set which will take care..

Reply:
Fixed.